backend/src/chats/chat/chat.controller.ts

```diff
@@ -12,14 +12,18 @@ import { ChatService } from './chat.service';
 import { CreateChatDto } from './dto/create-chat.dto';
 import { UpdateChatDto } from './dto/update-chat.dto';
 import { Chat } from './entities/chat.entity';
+import * as bcrypt from 'bcrypt';
 
 @Controller('chats')
 export class ChatController {
 	constructor(private readonly chatService: ChatService) {}
 
 	@Post()
-	create(@Body() createChatDto: CreateChatDto) {
-		// use bcrypt to hash passwords!
+	async create(@Body() createChatDto: CreateChatDto) {
+		if (createChatDto.hasOwnProperty('password')) {
+			const hashed = await bcrypt.hash(createChatDto.password, 11);
+			createChatDto.password = hashed;
+		}
```

Comment:
Same as  #102 not possible to create a chat without a password, in DTO = optional in entity mandatory. 
Do we want to be able to make chats without a pw?  If so we can add it as an optional in the entity.

Reply:
this is a better spot for this xD But I had already responded in the other PR as well.
but copy pasta here for completeness sake

Changing the entity password to @Column({ nullable: true }) solves this

Reply:
alright!